app/components/PartialChallengeDetailsView.js

```diff
@@ -91,15 +66,51 @@ PartialChallengeDetailsView.propTypes = {
 }
 
 const styles = StyleSheet.create({
```

Comment:
I broke this out into a separate file and then imported that file. That way the styles are more compartmentalized. You can check it out by looking at the ChallengeCreationScene file on my branch. We should discuss this as a group and see which pattern we want to follow so that we have a more uniform code base. @jwt1143 thoughts?